tasks/create.yml

```diff
@@ -51,20 +59,25 @@
 
 - name: Stop and delete server and client service
   block:
+  - name: Probe installed services
+    service_facts:
   - name: Ensures BeeGFS client service is not running if force_format is requested
     service:
       name: beegfs-client
       state: stopped
     become: yes
     when:
+    - "'beegfs-client.service' in ansible_facts.services"
     - beegfs_enable.oss | bool
     - beegfs_enable.client | bool
   - name: Stop BeeGFS server services
     service:
       name: "{{ item.name }}"
       state: stopped
     become: yes
-    when: item.enable | bool
+    when:
+    #- item.name in ansible_facts.services
```

Comment:
This one needs fixing because the systemd server units have some uniqueness from (in this case) port number in the service name.

Reply:
Should this be uncommented?